mangaki/mangaki/wrappers/anilist.py

```diff
@@ -425,3 +468,81 @@ def get_user_list(self,
     getattr(settings, 'ANILIST_CLIENT', None),
     getattr(settings, 'ANILIST_SECRET', None)
 )
+
+def insert_works_into_database_from_anilist(entries: List[AniListEntry]) -> Optional[List[Work]]:
+    """
+    Insert works into Mangaki database from AniList data, and return Works added.
+    Alternative WorkTitle are also added to the database, as well as a Reference.
+    :param entries: a list of entries from AniList to insert if not present in the database
+    :type entries: List[AniListEntry]
+    :return: a list of works effectively added in the Mangaki database
+    :rtype: Optional[List[Work]]
+    """
+    new_works = []
+
+    for entry in entries:
+        titles = {synonym: ('unknown', 'synonym') for synonym in entry.synonyms}
+        titles.update({entry.title: ('romaji', 'official')})
+        titles.update({entry.english_title: ('english', 'main')})
+        titles.update({entry.japanese_title: ('japanese', 'official')})
+
+        anime_type = entry.media_type if entry.work_type == AniListWorks.animes else ''
+        manga_type = entry.media_type if entry.work_type == AniListWorks.mangas else ''
+
+        studio = None
+
+        category = (
+            work_categories.anime if entry.work_type == AniListWorks.animes else
```

Comment:
Indeed, that's way cleaner and slightly easier to read !
I'll switch to something like this in my next commit 👍